var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -149,6 +149,14 @@ class Openblas(MakefilePackage):
 
     depends_on('perl', type='build')
 
+    def flag_handler(self, name, flags):
+        spec = self.spec
+        iflags = []
+        if name == 'ldflags':
+            if spec.satisfies('@0.3.20 %oneapi'):
+                iflags.append('-lifcore')
+        return (iflags, None, flags)
```

Comment:
```suggestion
        if name == 'ldflags':
            if spec.satisfies('@0.3.20 %oneapi'):
                flags.append('-lifcore')
        return (flags, None, None)
```
With this suggestion, this would mean "keep current behavior and add the `-lifcore` flag.

As a `MakefilePackage`, openblas cannot take advantage of the third return value for a flag handler.